Objects/boolobject.c

```diff
@@ -1,26 +1,20 @@
 /* Boolean type, a subtype of int */
 
 #include "Python.h"
+#include "pycore_runtime.h"       // _Py_ID()
 #include "pycore_pyerrors.h"      // _Py_FatalRefcountError()
 
 /* We define bool_repr to return "False" or "True" */
 
-static PyObject *false_str = NULL;
-static PyObject *true_str = NULL;
-
 static PyObject *
 bool_repr(PyObject *self)
 {
-    PyObject *s;
-
-    if (self == Py_True)
-        s = true_str ? true_str :
-            (true_str = PyUnicode_InternFromString("True"));
-    else
-        s = false_str ? false_str :
-            (false_str = PyUnicode_InternFromString("False"));
-    Py_XINCREF(s);
-    return s;
+    if (self == Py_True) {
+        return &_Py_ID(True);
+    }
+    else {
+        return &_Py_ID(False);
+    }
```

Comment:
Wouldn't ternary operator be more compact here?

```suggestion
    return (self == Py_True) ? &_Py_ID(True) : &_Py_ID(False);
```

Reply:
You bet.  Mostly I was matching the structure of the existing code.  However, I don't mind a little cleanup like this.